examples/javamail/README_pt.adoc

```diff
@@ -0,0 +1,201 @@
+:index-group: Unrevised
+:jbake-type: page
+:jbake-status: status=published
```

Comment:
I would suggest chaning this line to:
`:jbake-status: published`

Reply:
fixed